### PR TITLE
DKS AMI - first commit

### PR DESCRIPTION
**Pull request by tea-short**

### CODE REVIEW

File: dks-host/dks-host-install.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+# Install Java
+sudo yum update -y
+sudo yum install -y java-1.8.0-openjdk-devel
+
+# Install Amazon SSM agent
+sudo yum install -y https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/linux_amd64/amazon-ssm-agent.rpm
+
+# Download & install latest DKS service artifact
+curl -s https://api.github.com/repos/dwp/data-key-service/releases/latest \
```
Comment (mattburgess) [inline dks-host/dks-host-install.sh]: This is much easier (for me!) to understand as `curl -s https://api.github.com/repos/dwp/data-key-service/releases/latest | jq -r '.assets[0].browser_download_url' | xargs wget --no-check-certificate -O /tmp/dks.jar` but I'm not sure if CentOS installs `jq` by default?

File: dks-host/dks [hunk 2]
```diff
@@ -0,0 +1,105 @@
+#!/bin/sh
```
Comment (mattburgess) [inline dks-host/dks]: systemd unit files are generally much easier to read & write, although I must admit to having very little experience with them myself. May be worth a future ticket to explore replacing this old-style init script with a systemd unit file. Or perhaps see if Mark/Steve have any experience to replace this now?

Comment (tea-short) [reply dks-host/dks]: We're following the pattern of Kafka broker here. Happy to explore systemd option in a future ticket but the current pattern will give us a quick win - what do you think?

File: dks-host/dks-host-install.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
+#!/bin/sh -x
+
+## Script to install DKS service
+
+# Install Java
+sudo yum update -y
+sudo yum install -y java-1.8.0-openjdk-devel
```
[the remark this answers is not in the document]
Comment (tea-short) [reply dks-host/dks-host-install.sh]: @DanielChicot Hey Dan, is this the correct Java version for DKS? We're going off what's in readme.md

File: dks-host/dks-host-install.sh (hunk 1 above)
Comment (tea-short) [reply dks-host/dks-host-install.sh]: No jq there...

File: dks-host/dks-host-install.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (DanielChicot) [reply dks-host/dks-host-install.sh]: > @DanielChicot Hey Dan, is this the correct Java version for DKS? We're going off what's in readme.md

Hi Dmitrijs - I can confirm that java-1.8.0-openjdk-devel is the correct version.

[the remark this answers is not in the document]
Comment (tea-short) [reply dks-host/dks-host-install.sh]: Great, thanks Dan!

**Pull request merged**